Documentation/LogFileManagement.md

```diff
@@ -47,20 +47,19 @@ Let's take a look at the DDLogFileManager protocol:
 
 // Private methods (only to be used by DDFileLogger)
 
-- (NSString *)createNewLogFile;
+- (NSString *)createNewLogFileWithError(NSError**)error;
 
 @optional
 
 // Notifications from DDFileLogger
 
-- (void)didArchiveLogFile:(NSString *)logFilePath;
-- (void)didRollAndArchiveLogFile:(NSString *)logFilePath;
+- (void)didRollAndArchiveLogFile:(NSString *)logFilePath wasRolled:(BOOL)wasRolled;
```

Comment:
Correction: (void)**didArchiveLogFile**:(NSString *)logFilePath wasRolled:(BOOL)wasRolled;

Reply:
Ah, copied the wrong method. Thanks!